bittensor/utils/__init__.py

```diff
@@ -66,7 +66,7 @@ def version_checking():
     latest_version_as_int = (100 * int(version_split[0])) + (10 * int(version_split[1])) + (1 * int(version_split[2]))
 
     if latest_version_as_int > bittensor.__version_as_int__:
-        print('\u001b[31m Current Bittensor Version: {}, Latest Bittensor Version {} \n Please update to the latest version'.format(bittensor.__version__,latest_version))
+        print('\u001b[31mBittensor Version: Current {}/Latest {}\nPlease update to the latest version\u001b[0m'.format(bittensor.__version__,latest_version))
```

Comment:
Should we let them know that they don't need to do it urgently?

Reply:
I think if we make it yellow maybe.
And yes we can add like `"Please update your client when convenient"`

Reply:
@eduardogr how's this?